source/leuven/wice_quick_start.rst

```diff
@@ -62,7 +62,7 @@ a valid value.
 To submit to a compute node you need to provide the required number of nodes and cores. 
 For example to request 2 nodes with each 72 cores for 2 hours you can submit like this::
 
-   $ sbatch --cluster=wice --nodes=2 --ntasks-per-node=72 --time=2:00:00  -A lp_myproject  myjobscript.slurm
+   $ sbatch --account=lp_myproject --cluster=wice --nodes=2 --ntasks-per-node=72 --time=2:00:00 myjobscript.slurm
```

Comment:
I was also confused that `--cluster=` works eventhough the Slurm documentation only mentions `--clusters=`. So, I only made this change for compliance with Slurm docs.

Reply:
You can indeed truncate long-format arguments; Slurm will only complain if it becomes unclear what should be the matching argument, for example:
```bash
# This works
srun --par=batch ...
# This fails
srun --pa=batch ...
srun: option '--pa=batch' is ambiguous; possibilities: '--pack-group' '--partition'
```
Applying this to the `--cluster/clusters` case,  Slurm will actually suggest both possibilities:
```bash
srun --cl=genius ...
srun: option '--cl=genius' is ambiguous; possibilities: '--cluster-constraint' '--clusters' '--cluster'
```
So I think there is something to say for both `--cluster` and `--clusters` . Ehsan: as far as I'm concerned you can change it here, but then please also ensure that all other documentation (e.g. HPCIntro) follows the same choice. At first sight the other VSC sites are not mentioning this option because they don't need it (either because of how cluster modules are used or how login nodes are set up), so no possible conflicts there.

Reply:
OK -- I just found two places where `--cluster` has not yet been replaced with `--clusters`:
wice advanced guide: L109
wice quick start: L60

Reply:
Correct! I just fixed both of them. Thanks for the catch.